project_templates/technote_latex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/Makefile

```diff
@@ -13,13 +13,23 @@ endif
 
 export TEXMFHOME ?= lsst-texmf/texmf
 
+# Add aglossary.tex as a dependancy here if you want a glossary
 $(DOCNAME).pdf: $(tex) meta.tex local.bib
 	latexmk -bibtex -xelatex -f $(DOCNAME)
+%	makeglossaries $(DOCNAME)      
```

Comment:
This is the wrong comment character, I think.

Reply:
indeed